clients/spark/build.sbt

```diff
@@ -89,9 +89,15 @@ lazy val commonSettings = Seq(
   version := projectVersion
 )
 
+val nexus = "https://s01.oss.sonatype.org/"
 lazy val publishSettings = Seq(
+  publishResolvers := Seq(
+    if (isSnapshot.value)
+      MavenRepository("lakefs-snapshots", s"$nexus/content/repositories/snapshots")
+    else MavenRepository("lakefs-releases", s"$nexus/service/local/staging/deploy/maven2"),
+    MavenRepository("lakefs-s3-repo", "s3://treeverse-clients-us-east/"),
+  ),
   publishTo := {
-    val nexus = "https://s01.oss.sonatype.org/"
     if (isSnapshot.value) Some("snapshots" at nexus + "content/repositories/snapshots")
```

Comment:
It is unfortunate that we need to say "snapshots" here and in the resolver settings.

Reply:
Yes, although the `sbt-publish-more` should set the publishTo by default, which would remove the need to explicitly specify it - it doesn't work well with subprojects.